client/src/components/organisms/TimeLine/TimeLine.jsx

```diff
@@ -0,0 +1,149 @@
+import React from 'react';
+import { Container } from 'react-bootstrap';
+import './TimeLine.css';
+
+import IcoMoon from 'react-icomoon';
+
+const TimeLine = () => {
+  return (
+    <Container>
+      <ul className="cbp_tmtimeline">
+        <li>
+          <time className="cbp_tmtime" dateTime="2013-04-10 18:30">
+            <span>4/10/13</span> <span>18:30</span>
+          </time>
+          {/* <div className="cbp_tmicon cbp_tmicon-phone" /> */}
+          <IcoMoon icon="phone"></IcoMoon>
+          <div className="cbp_tmlabel">
```

Comment:
classname convention: which one is followed?
a) class_name
b)className
c)class-name
found few variations across loginmodal jsx, test files

Reply:
The class names are different because they all come from different sources.